### PR TITLE
add support for the DescribeSecret restful API

### PR DESCRIPTION
**Pull request by wanghouqi**
this restful API will be exposed to public as:
 POST <ehsm_srv_address>/ehsm?Action=DescribeSecret

 Obtains the metadata of a secret.

 test: passed the python test

Signed-off-by: wanghouqi <wanghouqi@vip.qq.com>

### CODE REVIEW

File: ehsm_kms_service/secret_manager_apis.js [hunk 1]
```diff
@@ -265,9 +265,57 @@ const createSecret = async (res, appid, payload, DB) => {
...
+            if (doc['plannedDeleteTime']) {
+                result['plannedDeleteTime'] = doc['plannedDeleteTime']
+            }
+            if (doc['rotationInterval']) {
+                result['rotationInterval'] = doc['rotationInterval']
+                result['lastRotationDate'] = doc['lastRotationDate']
+                result['nextRotationDate'] = doc['nextRotationDate']
+            }
+        }
+
```
Comment (yang8621) [inline ehsm_kms_service/secret_manager_apis.js:305]: else?

Comment (wanghouqi) [reply ehsm_kms_service/secret_manager_apis.js:305]: There is no else. If it cannot be found, an empty JSON will be returned

File: ehsm_kms_service/secret_manager_apis.js [hunk 2]
```diff
@@ -265,9 +299,75 @@ const createSecret = async (res, appid, payload, DB) => {
...
+        // search secret_metadata 
+        const query = {
+            selector: {
+                appid,
+                secretName
+            },
+            fields: ['secretName', 'description', 'createTime', 'plannedDeleteTime', 'rotationInterval', 'lastRotationDate', 'nextRotationDate'],
+            limit: 1,
```
Comment (yang8621) [inline ehsm_kms_service/secret_manager_apis.js:337]: which version of the secret will be returned?

**Comment by wanghouqi** [comment]
Move to https://github.com/intel/ehsm/pull/155

**Pull request closed without merging**